stations.csv

```diff
@@ -3115,7 +3115,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 4085;Lus-la-Croix-Haute;lus-la-croix-haute;8774766;87747667;44.6709898;5.6973678;;FR;Europe/Paris;f;f;f;t;f;f;FRHWK;LCX;t;;f;8703862;f;;f;;f;;f;;f;;f;8703862;f;;f;;f;;;;;f;;f;8700376;f;;f;FRHWK;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4085;
 4088;Moirans—La-Galifette;moirans-la-galifette;8774769;87747691;45.324154;5.565170;4774;FR;Europe/Paris;f;f;f;t;f;f;FRHWN;MFE;t;;f;8704001;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv4088;
 4089;Genève Voyageurs;geneve-voyageurs;8774890;87748905;46.209768;6.140756;5335;CH;Europe/Zurich;f;f;f;f;f;f;FRHWO;;t;;f;8774890;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv4089;
-4090;Genève;geneve;8774893;87748939;;;5335;CH;Europe/Zurich;f;f;f;f;f;f;FRHWQ;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv4090;
+4090;Genève;geneve;8501008;8501008;;;5335;CH;Europe/Zurich;f;f;f;f;f;f;FRHWQ;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv4090;
```

Comment:
I can confirm the reference code for the city from ERA's [reference data portal](https://teleref.era.europa.eu/). ![image](https://github.com/trainline-eu/stations/assets/4478372/d71901a9-022f-47ec-b3fb-de73be41f0dd)

However the UIC-8 field is supposed to contain 8 digits (what a surprise!). After some digging, I found this [publicly accessible document](https://www.era.europa.eu/system/files/2022-11/era_technical_document_tap_b_9.pdf), that lays out how it's computed.
![image](https://github.com/trainline-eu/stations/assets/4478372/5dd859b7-7711-4de5-8e49-7da44377d9cb) 
![image](https://github.com/trainline-eu/stations/assets/4478372/8068fb49-bfd9-4454-8f13-5a8548633fdc)